### DISCUSSION TITLE
klipper version

### DISCUSSION THREAD

**Discussion by jlodew0** [xblax/flashforge_ad5m_klipper_mod#3] [Q&A]
Since FF just seem to have uploaded a generic klipper source and some binaries. (probably to comply (sort of) to the gpl license)
Are we sure that this is the version used on the 5m series?
I was wondering a bit around for the mcu (n32g455) and was surprised to see that support was only added since klipper 12.0

I've wrote my observation here:
https://github.com/g992/flashforge-ad5m-5mpro-research/issues/8

**Comment by xblax** [comment]
Yes, see my comment in that issue. Using a new Klipper does not work, it causes issue with the MCUs because there are some changes in the protocol. I don't want to look info flashing MCUs in the moment. This would also have to be figured first. For now, I want to keep the mod compatible with the stock software anyway. Klipper 0.11 has most of the relevant features.

**Comment by g992** [comment]
just like that, it will not work to assemble the klipper by specifying h32 in it, at work we tried to install h32 instead of stm32, but it did not work :)
the controller simply does not boot
, but if you assemble the firmware for stm32f103, the controller will boot, but analog peripherals will not be available
